### PR TITLE
add media facade config

### PR DESCRIPTION
**Pull request by Thiblef**
https://github.com/open-orchestra/open-orchestra-base-api-bundle/pull/67
https://github.com/open-orchestra/open-orchestra-cms-bundle/pull/1288
https://github.com/open-orchestra/open-orchestra-workflow-function-bundle/pull/73

### CODE REVIEW

**Comment by nicolasThal** [comment]
:+1:

**Comment by itkg-ngilain** [comment]
:+1: but beware to rebase !

File: MediaAdminBundle/Transformer/MediaCollectionTransformer.php [hunk 1]
```diff
@@ -23,7 +22,7 @@ class MediaCollectionTransformer extends AbstractSecurityCheckerAwareTransformer
      */
     public function transform($mixed, $folderId = null, $folderDeletable = false, $parentId = null)
     {
-        $facade = new MediaCollectionFacade();
+        $facade = $this->newFacade();
```
Comment (itkg-ngilain) [inline MediaAdminBundle/Transformer/MediaCollectionTransformer.php]: Can you add an exception ``if (!$facade instanceof FacadeInterface)``

**Comment by ngilain** [comment]
:+1: but beware of the rebase

File: MediaAdminBundle/DependencyInjection/Configuration.php [hunk 2]
```diff
@@ -70,6 +70,23 @@ public function getConfigTreeBuilder()
                     ->end()
                 ->end()
             ->end()
+            ->arrayNode('transformer')
```
Comment (alavieille) [inline MediaAdminBundle/DependencyInjection/Configuration.php]: same remark than in cms bundle

**Comment by Thiblef** [comment]
@itkg-alavieille @nicolasThal @itkg-ngilain

**Comment by alavieille** [comment]
:+1:

**Comment by ngilain** [comment]
:+1:

File: MediaAdminBundle/Tests/DependencyInjection/Fixtures/config/value.yml [hunk 3]
```diff
@@ -13,3 +13,6 @@ open_orchestra_media_admin:
                     max_width: 5000
                     max_height: 5000
                     compression_quality: 10000
+    facades:
+        media:             FacadeClass
+        media_collection:  FacadeClass
```
Comment (alavieille) [inline MediaAdminBundle/Tests/DependencyInjection/Fixtures/config/value.yml:18]: eof

**Comment by Thiblef** [comment]
@itkg-alavieille

**Comment by alavieille** [comment]
:+1:

**Pull request merged**